model/host/host_test.go

```diff
@@ -1050,3 +1050,395 @@ func TestInactiveHostCountPipeline(t *testing.T) {
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostProvisioning,
+	}
+	host7 := &Host{
+		Id:          "host7",
+		Distro:      distro.Distro{Id: d2},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task3",
+	}
+	host8 := &Host{
+		Id:     "host8",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostRunning,
+	}
+	assert.NoError(host1.Insert())
+	assert.NoError(host2.Insert())
+	assert.NoError(host3.Insert())
+	assert.NoError(host4.Insert())
+	assert.NoError(host5.Insert())
+	assert.NoError(host6.Insert())
+	assert.NoError(host7.Insert())
+	assert.NoError(host8.Insert())
+
+	containers, err := FindAllContainers()
+	assert.NoError(err)
+	assert.Equal(5, len(containers))
+}
+
+func TestFindAllContainersEmpty(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(Collection))
+
+	const d1 = "distro1"
+	const d2 = "distro2"
+
+	host1 := &Host{
+		Id:          "host1",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task",
+	}
+	host2 := &Host{
+		Id:     "host2",
+		Distro: distro.Distro{Id: d1},
+		Status: evergreen.HostStarting,
+	}
+	host3 := &Host{
+		Id:     "host3",
+		Distro: distro.Distro{Id: d1},
+		Status: evergreen.HostTerminated,
+	}
+	host4 := &Host{
+		Id:          "host4",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task2",
+	}
+	host5 := &Host{
+		Id:     "host5",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostProvisioning,
+	}
+	host6 := &Host{
+		Id:     "host6",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostProvisioning,
+	}
+	host7 := &Host{
+		Id:          "host7",
+		Distro:      distro.Distro{Id: d2},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task3",
+	}
+	host8 := &Host{
+		Id:     "host8",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostRunning,
+	}
+	assert.NoError(host1.Insert())
+	assert.NoError(host2.Insert())
+	assert.NoError(host3.Insert())
+	assert.NoError(host4.Insert())
+	assert.NoError(host5.Insert())
+	assert.NoError(host6.Insert())
+	assert.NoError(host7.Insert())
+	assert.NoError(host8.Insert())
+
+	containers, err := FindAllContainers()
+	assert.NoError(err)
+	assert.Equal(0, len(containers))
+}
+
+func TestGetContainers(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(Collection))
+
+	const d1 = "distro1"
+	const d2 = "distro2"
+
+	host1 := &Host{
+		Id:          "host1",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task",
+		IsParent:    true,
+		ParentID:    "parentId",
+	}
+	host2 := &Host{
+		Id:          "host2",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostStarting,
+		ContainerID: "containerId2",
+		ParentID:    "parentId1",
+	}
+	host3 := &Host{
+		Id:          "host3",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostTerminated,
+		ContainerID: "containerId3",
+		ParentID:    "parentId",
```

Comment:
This should be "host1", and same for below.

Reply:
done